cartographer/mapping_2d/sparse_pose_graph.cc

```diff
@@ -429,10 +428,8 @@ void SparsePoseGraph::AddNodeFromProto(const int trajectory_id,
         constant_data->gravity_alignment.inverse());
     optimization_problem_.AddTrajectoryNode(
         node_id.trajectory_id, constant_data->time,
-        transform::Project2D(constant_data->initial_pose *
-                             gravity_alignment_inverse),
-        transform::Project2D(pose * gravity_alignment_inverse),
-        constant_data->gravity_alignment);
+        constant_data->initial_pose * gravity_alignment_inverse,
```

Comment:
Remove the `gravity_alignment_inverse` here and on the next line?

Reply:
Done